lms/djangoapps/experiments/flags.py

```diff
@@ -56,14 +62,23 @@ def test_my_experiment(self):
             ...
 
     """
-    def __init__(self, waffle_namespace, flag_name, num_buckets=2, experiment_id=None, **kwargs):
+    def __init__(
+            self,
+            waffle_namespace,
+            flag_name,
+            num_buckets=2,
+            experiment_id=None,
+            use_course_aware_bucketing=True,
```

Comment:
This may have been answered, but I can't quite figure it out... what happens if someone calls `is_enabled` without a `course_key` and has left `use_course_aware_bucketing` to `True`?

It looks like we ignore the `use_course_aware_bucketing` value and just bucket them without the course_key.  Which makes sense, I think, but sorta wanted to make sure...

Reply:
Yup, you understand that correctly. I tried to describe that here: https://github.com/edx/edx-platform/pull/24718/files#diff-389abe6cfe6f33572b4e99c16dd38390R38-R40

Do you think that docstring is clear enough, or should I revisit it?

Reply:
My question above is definitely addressed, but perhaps implicitly if you know that `is_enabled` calls `get_bucket` :) 

While they're both public methods, I'd guess folks are more likely to interact with `is_enabled` than `get_bucket` directly, which was maybe my hangup.  It may be worth improving the doc a bit to explain what happens when you "use" the flag from outside, you know?

Reply:
Agreed! I'll make that change before merging.